.generation/generate.py

```diff
@@ -174,6 +174,7 @@ def generate_python_code(*, package_name: str, package_version: str, package_url
         ],
         check=True,
     )
+    shutil.rmtree(Path("python") / "docs")
```

Comment:
why did you add this?

Reply:
Because at the moment in the `main`branch there are no docs, so I thought they are not wanted in the package. But the generator outputs docs so I delete them.